### PR TITLE
Added project folder!

### LINKED ISSUE #175: Wallpaper Changer
## Wallpaper Changer

**Automate the process of changing your desktop wallpaper daily (with an element of randomness!) at a scheduled time of your choice.**

## Instructions

- Create a new folder for your script and file/folder name should be appropriate.
- Create a README.md in your folder for program Instructions
- add requirements.txt if needed

Please add/delete options that are not relevant.

- [x] Adding New Code

### Programming Language

- [x] Python

:star2: Star it :fork_and_knife:Fork it :handshake: Contribute to it!

Discord server  - https://discord.gg/FXyh2S3

Happy Coding,

### PR DESCRIPTION
**Pull request by SPOCTONE**
linked with issue #175

### CODE REVIEW

File: wallpaper-changer/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Dependencies:-
+
+- Schedule Library
+  Installation:
+  `pip install schedule`
+
+
```
Comment (codeperfectplus) [inline wallpaper-changer/requirements.txt]: improve it.

**Review by codeperfectplus — CHANGES_REQUESTED** [review]
there are too many jpeg and png image, please remove them

File: wallpaper-changer/requirements.txt (hunk 1 above)
Comment (codeperfectplus) [reply wallpaper-changer/requirements.txt]: use python standard requirements.txt way. check requirement.txt file example below.
 
https://github.com/uber/Python-Sample-Application/blob/master/requirements.txt

**Comment by SPOCTONE** [comment]
ok. let me fix the linting and i'll do as you said.

**Comment by SPOCTONE** [comment]
please check now.

**Comment by codeperfectplus** [comment]
still, requirements.txt is not improved.

**Comment by SPOCTONE** [comment]
What can I improve there in your opinion?

**Comment by SPOCTONE** [comment]
sorry read the above message just now 😅

**Review by codeperfectplus — APPROVED** [review]
use pip command to save requitements.txt so every time new user quickly install all requirements.txt by the terminal.
to save requirements.txt -                                   `pip freeze > requirements. txt`
to install the module from requirements.txt -       `pip install - requirements.txt`

Thanks for making your contributions

**Comment by SPOCTONE** [comment]
no problem! also thanks for the great advice and being patient with me ☺

**Pull request merged**